vpr/src/route/check_route.cpp

```diff
@@ -866,6 +887,30 @@ void check_net_for_stubs(const Netlist<>& net_list,
     }
 }
 
+static bool chanxy_chanz_adjacent(RRNodeId chanxy_node, RRNodeId chanz_node) {
```

Comment:
Move to rr_graph_utils or something (or make externally visible and call from nodes_are_adjacent).